### PR TITLE
Create test for hibernate search

### PR DESCRIPTION
**Pull request by benjamin-confino**

### CODE REVIEW

**Comment by benjamin-confino** [comment]
#build

**Comment by benjamin-confino** [comment]
#build

**Comment by benjamin-confino** [comment]
#build

File: dev/com.ibm.ws.cdi.third.party_fat/test-applications/hibernateSearchTest.war/src/cdi/hibernate/test/web/SimpleTestServlet.java [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+        pw.println("<h1>JPA Simple Test Case</h1>");
+        pw.println("<hr>");
+        pw.println("field bridge called: " + fieldBridgeCalled);
+        
+        try {
+            // The code that performs the create and query operations have been moved to CommonTestCode in order
+            // to share it with ActionedTestServlet.  You can continue with that pattern, or place all of
+            // your action's logic here.
+            CommonTestCode.populate(request, response, em, tx);
+            CommonTestCode.query(request, response, em, tx);
```
Comment (Azquelt) [inline dev/com.ibm.ws.cdi.third.party_fat/test-applications/hibernateSearchTest.war/src/cdi/hibernate/test/web/SimpleTestServlet.java]: The test that calls this servlet doesn't appear to validate whether this section succeeds or not. Is that intentional?

Comment (Azquelt) [reply dev/com.ibm.ws.cdi.third.party_fat/test-applications/hibernateSearchTest.war/src/cdi/hibernate/test/web/SimpleTestServlet.java]: I also don't see the referenced `ActionedTestServlet` anywhere

Comment (benjamin-confino) [reply dev/com.ibm.ws.cdi.third.party_fat/test-applications/hibernateSearchTest.war/src/cdi/hibernate/test/web/SimpleTestServlet.java]: It's intentional. What I'm really testing is if the app even starts up correctly.

**Comment by benjamin-confino** [comment]
#build

**Comment by benjamin-confino** [comment]
The final push was just uncommenting @Mode(TestMode.FULL)

**Pull request merged**